cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -134,8 +134,6 @@ if (EMSCRIPTEN_FORCE_COMPILERS)
 	set(CMAKE_CXX_COMPILER_ID Clang)
 	set(CMAKE_CXX_STANDARD_COMPUTED_DEFAULT 98)
 
-	set(CMAKE_C_COMPILER_TARGET "asmjs-unknown-emscripten")
```

Comment:
@jgravelle-google : This is ok to remove, as long as tests `other.test_cmake*` pass after this change. Was this verified?

The only visible difference to this change is I believe that these fields will come out empty to user CMakeLists.txt files, so user projects will see these being absent for them. This should not be a big issue though, the origin of these variables in CMake was to be used by toolchain file implementors, and not by CMake project build script writers.